assets/timber.js.liquid

```diff
@@ -127,6 +124,19 @@ timber.accessibleNav = function () {
   }
 };
 
+timber.drawersInit = function () {
+  timber.LeftDrawer = new timber.Drawers('NavDrawer', 'left');
```

Comment:
open question, is the mobile drawer a pattern we'd like to encourage for all mobile-ready themes?
cc @pauldpritchard